EssentialFeed/EssentialFeediOSTests/FeedCommentsSnapshotTests.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  FeedCommentsSnapshotTests.swift
+//  EssentialFeediOSTests
+//
+//  Created by Danil Vassyakin on 4/19/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+@testable import EssentialFeed
+import EssentialFeediOS
+
+class FeedCommentsSnapshotTests: XCTestCase {
+
+	func test_emptyComments() {
+		let sut = makeSUT()
+
+		sut.display(noComments())
+
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light)), named: "FEED_IMAGE_COMMENTS_EMPTY_light")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark)), named: "FEED_IMAGE_COMMENTS_EMPTY_dark")
+		
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light, size: .extraExtraExtraLarge)), named: "FEED_IMAGE_COMMENTS_EMPTY_LARGE_light")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark, size: .extraExtraExtraLarge)), named: "FEED_IMAGE_COMMENTS_EMPTY_LARGE_dark")
+	}
+
+	func test_commentsWithErrorMessage() {
+		let sut = makeSUT()
+
+		sut.display(errorMessage("This is\na multi-line\nerror message"))
+		
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light)), named: "FEED_IMAGE_COMMENTS_ERROR_light")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark)), named: "FEED_IMAGE_COMMENTS_ERROR_dark")
+		
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light, size: .extraExtraExtraLarge)), named: "FEED_IMAGE_COMMENTS_ERROR_LARGE_light")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark, size: .extraExtraExtraLarge)), named: "FEED_IMAGE_COMMENTS_ERROR_LARGE_dark")
```

Comment:
Avoid creating unnecessary snapshot tests because they're expensive to create/compare and can slow down the CI pipeline.

Checking a Dynamic Type with a single style is enough.

Remove one of the assertions and its snapshot file.

```suggestion
		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light, size: .extraExtraExtraLarge)), named: "FEED_IMAGE_COMMENTS_ERROR_LARGE_light")
```